graphile-build is missing chalk dependency declaration

When using the production installation with graphile-build somewhere in the dependency tree, postgrahile does not init properly:
```
Error: Cannot find module 'chalk'
     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:580:15)
     at Function.Module._load (internal/modules/cjs/loader.js:506:25)
     at Module.require (internal/modules/cjs/loader.js:636:17)
     at require (internal/modules/cjs/helpers.js:20:18)
     at Object.<anonymous> (/.../node_modules/graphile-build/node8plus/extend.js:9:14)
```
If I'm not mistaken, graphile-build depends on chalk directly, e.g. here:
https://github.com/graphile/graphile-engine/blob/master/packages/graphile-build/src/extend.js#L2

However, the package.json for grahile-build does not list chalk as a dependency:
https://github.com/graphile/graphile-engine/blob/master/packages/graphile-build/package.json#L32

This does not fail in dev mode, probably because the dev dependency jest includes a dependency on chalk itself.

--- a/packages/graphile-build/package.json
+++ b/packages/graphile-build/package.json
@@ -31,6 +31,7 @@
   "homepage": "https://graphile.org/graphile-build/",
   "dependencies": {
     "@types/graphql": "^0.13.4",
+    "chalk": "^2.1.0",
     "debug": ">=2 <3",
     "graphql-parse-resolve-info": "4.0.0",
     "lodash": ">=4 <5",

